udf-compiler/src/test/scala/com/nvidia/spark/OpcodeSuite.scala

```diff
@@ -59,5 +55,1830 @@ class OpcodeSuite extends FunSuite {
     ds1.show
     ds2.show
   }
-}
 
+  // START OF TESTS
+
+  test("not boolean") {
+    val myudf: Boolean => Boolean = { x => !x }
+    val u = makeUdf(myudf)
+    val dataset = List(true, false, true, false).toDS().repartition(1)
+    val result = dataset.withColumn("new", u('value))
+    val equiv = dataset.withColumn("new", not(col("value")))
+    result.show
```

Comment:
Some tests have `.show` each time. I can see that being useful for debugging, and if we find a test failure we can augment the tests, but I am not sure we want this every time the tests run. Could we remove the `.show` and ensure the tests run without extra output for the CI runs?

Reply:
Removed.